src/sentry/static/sentry/app/views/settings/components/settingsNavItem.jsx

```diff
@@ -39,6 +40,15 @@ const StyledNavItem = styled(Link)`
   }
 `;
 
+const NewBadge = styled(Badge)`
```

Comment:
Do we need any of the CSS from `Badge`, design-wise, they seem pretty different

Reply:
I started down that path, but i think there were enough things... positioning, text size, props -> children stuff, that badge provides to make this route slightly less annoying.